### PR TITLE
feat: [part 02] adds loading indicator for the whole component

### PR DESCRIPTION
**Pull request by paschalidi**
Here I added a loading indicator for the whole page. Before the save buttons werent included in the loading which was odd.

### Before
image
### Now
image

### CODE REVIEW

File: src/core_modules/capture-core/components/DataEntries/EnrollmentRegistrationEntry/EnrollmentRegistrationEntry.component.js [hunk 1]
```diff
@@ -1,22 +1,32 @@
...
     marginTop: {
         marginTop: typography.pxToRem(2),
     },
 });
 
+const translatedTextWithStylesForProgram = (trackedEntityName: string, programName: string, orgUnitName: string) =>
+    (<>
+        {i18n.t('Saving a {{trackedEntityName}} in', { trackedEntityName })} <b>{programName}</b>
+        {orgUnitName && <>{' '}{i18n.t('in')} <b>{orgUnitName}</b></>}.
+    </>);
```
Comment (paschalidi) [inline src/core_modules/capture-core/components/DataEntries/EnrollmentRegistrationEntry/EnrollmentRegistrationEntry.component.js:27]: Shall we make a ticket for this @JoakimSM?

Comment (JoakimSM) [reply src/core_modules/capture-core/components/DataEntries/EnrollmentRegistrationEntry/EnrollmentRegistrationEntry.component.js:27]: Yeah, perfect if you make a ticket for this.

**Review by JoakimSM — APPROVED** [review]
LGTM 👍

File: src/core_modules/capture-core/components/DataEntries/EnrollmentRegistrationEntry/EnrollmentRegistrationEntry.component.js (hunk 1 above)
Comment (paschalidi) [reply src/core_modules/capture-core/components/DataEntries/EnrollmentRegistrationEntry/EnrollmentRegistrationEntry.component.js:27]: I did. Can you fill in the details please? 
https://jira.dhis2.org/browse/DHIS2-10267 I dont really know what was the problem on this one. I think you know best

**Pull request merged**